src/Modal.js

```diff
@@ -466,4 +485,7 @@ Modal.Header = Header;
 Modal.Title = Title;
 Modal.Footer = Footer;
 
+Modal.TRANSITION_DURATION = 300;
+Modal.BACKDROP_TRANSITION_DURATION = 150;
```

Comment:
I did it this way because that's how [bootstrap does it](https://github.com/twbs/bootstrap/blob/master/js/modal.js#L38). They are public, but only begrudgingly, and don't really control the duration of the animation (the css class does). it is just an unfortunate hack to get around the fact that transition end events in browsers just don't fire sometimes.